.gitignore

```diff
@@ -1,3 +1,4 @@
+/.hypothesis
```

Comment:
Can we configure this to go under `build/`?

Reply:
Sure. Two options for this, you can either set this in a .settings object along with any other required values, or in an HYPOTHESIS_DATABASE_FILE env variable. Whats your view?

Reply:
The former is preferable, we do the same for pytest (https://github.com/elifesciences/profiles/blob/develop/pytest.ini#L3).